Services/Comments/CommentService.cs

```diff
@@ -0,0 +1,128 @@
+﻿using Blog.Models;
+using Blog.Context;
+using Blog.ResponseExceptions;
+using System;
+using MySqlX.XDevAPI.CRUD;
+
+namespace Blog.Services.Comments
+{
+    public class CommentService:ICommentService
+    {
+        private readonly BlogDbContext _context;
+
+        public CommentService(BlogDbContext context)
+        {
+            _context = context;
+        }
+
+
+
+        private Comment findComment(int commentId)
+        {
+            var comment = _context.Comments.FirstOrDefault(c => c.Id == commentId);
+            if (comment == null)
+            {
+                throw new ResponseExceptions.BaseResponseException($"Comment with ID {commentId} does not exist.", ResponseExceptions.StatusCodes.NOT_FOUND);
```

Comment:
you could also directly use thet Not Found Exception thats inherting from ResponseException